### ISSUE TITLE
(PDF) SENTiVENT: Enabling Supervised Information Extraction of Company-Specific Events in Economic and Financial News

### ISSUE THREAD

**Issue by ShellLM** [irthomasthomas/undecidability#854]
- [ ] [(PDF) SENTiVENT: Enabling Supervised Information Extraction of Company-Specific Events in Economic and Financial News](https://www.researchgate.net/publication/354573583_SENTiVENT_Enabling_Supervised_Information_Extraction_of_Company-Specific_Events_in_Economic_and_Financial_News)

# (PDF) SENTiVENT: Enabling Supervised Information Extraction of Company-Specific Events in Economic and Financial News

## Snippet

"We present SENTiVENT, a corpus of fine-grained company-specific events in English economic news articles. The domain of event processing is highly productive and various general domain, fine-grained event extraction corpora are freely available but economically-focused resources are lacking. This work fills a large need for a manually annotated dataset for economic and financial text mining applications. A representative corpus of business news is crawled and an annotation scheme developed with an iteratively refined economic event typology. The annotations are compatible with benchmark datasets (ACE/ERE) so state-of-the-art event extraction systems can be readily applied. This results in a gold-standard dataset annotated with event triggers, participant arguments, event co-reference, and event attributes such as type, subtype, negation, and modality. An adjudicated reference test set is created for use in annotator and system evaluation. Agreement scores are substantial and annotator performance adequate, indicating that the annotation scheme produces consistent event annotations of high quality. In an event detection pilot study, satisfactory results were obtained with a macro-averaged F1-score of 59% validating the dataset for machine learning purposes. This dataset thus provides a rich resource on events as training data for supervised machine learning for economic and financial applications. The dataset and related source code is made available at [https://osf.io/8jec2/](https://osf.io/8jec2/)."

## Full Text

[https://www.researchgate.net/publication/354573583_SENTiVENT_Enabling_Supervised_Information_Extraction_of_Company-Specific_Events_in_Economic_and_Financial_News](https://www.researchgate.net/publication/354573583_SENTiVENT_Enabling_Supervised_Information_Extraction_of_Company-Specific_Events_in_Economic_and_Financial_News)

#### None
